asammdf/gui/ui/file_widget.ui

```diff
@@ -20,11 +20,11 @@
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
-      <number>0</number>
+      <number>2</number>
```

Comment:
The starting point should be to show the channel selection tab (index 0)

Reply:
canged